.github/workflows/integration-tests.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           coverage: "none"
-          extensions: "json,couchbase-3.2.2,memcached,mongodb-1.12.0,redis,rdkafka,xsl,ldap,relay"
+          extensions: "json,couchbase-3.2.2,memcached,mongodb-1.12.0,redis,rdkafka,xsl,ldap,relay,rdkafka"
```

Comment:
```rdkafka``` was already present in this string, it is now duplicated

Reply:
Whoops, fixed!